python/dgl/__init__.py

```diff
@@ -48,7 +47,6 @@
     set_node_lazy_features,
     set_src_lazy_features,
 )
-from .dataloading2 import *
```

Comment:
this results in CI failure, I think.

Reply:
I think it's better to update `dgl.ItemSet` in tests to `graphbolt.ItemSet` in this PR.

Reply:
fixed